deploy/crds/Keycloak.yaml

```diff
@@ -29,6 +29,28 @@ spec:
           type: object
         spec:
           type: object
+          properties:
+            externalDatabaseSecret:
+              description: 'Opotional. A Secret containing the connection string and credentials to an external database. If not provided a Postgres
+                instance will be created in the same namespace'
+              type: string
+            adminCredentialSecret:
+              description: 'Optional. The name of the Secret that contains the admin password in a `password` property. If not provided a password will be
+                auto generated and written to a Secret with the name `admin-credentials`'
+              type: string
+              required: false
+            extensions:
+              description: 'A list of extensions to be loaded by Keycloak on startup'
+              type: array
+              items:
+                type: object
+                properties:
+                  name:
+                    description: 'Extension name'
```

Comment:
There is currently no repository for keycloak extensions. I wonder, should we instead ask for a download URL?

Or use the old approach where we bundle the extensions with the init container?

Reply:
As discussed on the call, lets use the old approach for now. So using the old approach, what should be changed in the spec? To list the initcontainer image name and location maybe?

Reply:
For the old approach we only need the names of the extensions. The init container image and tag are hardcoded in the operator unless we want to make it configurable.

Reply:
ok, perfect. And the names of the extensions are used to determine what is added through the init container?

Reply:
> There is currently no repository for keycloak extensions. I wonder, should we instead ask for a download URL?

Currently there is no mechanism in Keycloak (nor RHSSO) images to download and install extensions on the fly. I'm not sure if there should be any (as this is a potential attack vector). So I think we should provide our own way of installing them. I'm OK with the old approach. Just wanted to check, what do you think about using Maven to download them?